internal/rollup/system.config.js

```diff
@@ -0,0 +1,10 @@
+(function(global) {
```

Comment:
👍 Don't want to put too much doc here since this file is loaded in the browser. I put more information about this file in the skydoc comment for additional_entry_points.